src/less/drawer/base/drawer.less

```diff
@@ -68,6 +68,7 @@
 }
 
 .drawer__window--expanded {
+    height: 95%;
     max-height: 95%;
```

Comment:
I'm fine with it, but what is the point of having both `height` and `max-height`

Reply:
Never mind, I remembered this is to override the other `max-height`

Reply:
I believe it prevents an override of `height: 100%`